paper.md

```diff
@@ -218,20 +218,21 @@ about is not clear to me.-->
 
 **Modularization is needed to scale.**
 Research workflows impose additional demands for an efficient research data management (RDM) platform besides "version control" and "data transport".
-Many research datasets contain millions of files, and that precludes placing such datasets in their entirety within a single Git repository even if individual files are tiny in their size.
+Many research datasets contain millions of files, and that precludes placing such datasets in their entirety within a single Git/git-annex repository even if individual files are tiny in their size.
 Such datasets should be partitioned into smaller subdatasets (e.g., a subdataset per each subject in the dataset comprising thousands of participants).
 This modularization allows for not only scalable management, <!-- unclear how management of 10k small pieces becomes more scalable than one piece with 10k parts; YOH: just by virtue of being able to handle 10k * 10k files --> but also for the efficient reuse of a selected subset of datasets.
 DataLad uses Git's submodule mechanism to unambiguously link (versions of) individual datasets into larger super-datasets, and further simplifies working with the resulting hierarchies of datasets with recursive operations across dataset boundaries.
 <!-- BEN: There are more aspect of modularization than reuse and large-scale.
 Most importantly, that's the notions of dependencies and a derivative
 relationsship that can be expressed that way, I think. -->
 <!-- YOH: well, there is "unambiguously link (versions of) " in above.  Could indeed be elaborated -->
-With this, DataLad makes it trivial to operate on individual files deep in the hierarchy or entire sub-trees of datasets, providing a "mono-repo"-like user experience in datasets nested arbitrarily deep.
+With this, DataLad makes it trivial to operate on individual files deep in the hierarchy or entire sub-trees of datasets, providing a "mono-repo"-like user experience in datasets which are nested arbitrarily deep.
 <!-- MIH: modularization is presented as #2, but it seems to be just #1 (nesting) described from a different angle -->
 <!-- YOH: not sure what #2 here, since in above it is "reproducible execution" in "MIH thinks:".
 
 As for #2 "reproducible execution" - I think we missed such subsection entirely, so I added it:
 --> 
+<!-- CM: Heading says: "Why Git and git-annex alone are not enough". This paragraph says "modulariyation is needed to scale" and  then describes how git-submodule supports modularizatoin (so, git is enough w.r.t. modulariztion!). Only then we describe what we add: i.e. a mono-repo-like experience, which is due to the recursive operations accross sub-modules. Should the title of the paragraph be something like: "Cross-module operations are required to scale"? Then a short sentence about "git supports nesting through the sub-module mechanism, we impleent recursive operation to use this mechanism to create a mono-repo experience. -->
```

Comment:
Agree. See point "A" in #64

Reply:
Thanks for pointing out issue #64, I missed that.